engine/src/main/java/org/destinationsol/game/SolCam.java

```diff
@@ -67,29 +63,24 @@ public void update(SolGame game) {
 
         Hero hero = game.getHero();
         float ts = game.getTimeStep();
+        life = hero.getLife();
         if (hero.isDead() || DIRECT_CAM_CONTROL) {
             applyInput(game);
-            life = hero.getLife();
-//        } else if (hero.isTranscendent()) {
-//            myPos.set(hero.getPosition());
-//        } else {
-//            Vector2 heroPos = hero.getHull().getBody().getWorldCenter();
-//            if (myZoom * VIEWPORT_HEIGHT < heroPos.dst(myPos)) {
-//                myPos.set(heroPos);
-//                game.getObjMan().resetDelays();
-//            } else {
-//                Vector2 moveDiff = SolMath.getVec(hero.getSpd());
-//                moveDiff.scl(ts);
-//                myPos.add(moveDiff);
-//                SolMath.free(moveDiff);
-//                float moveSpd = MOVE_SPD * ts;
-//                myPos.x = SolMath.approach(myPos.x, heroPos.x, moveSpd);
-//                myPos.y = SolMath.approach(myPos.y, heroPos.y, moveSpd);
-//            }
-//            life = hero.getLife();
         } else {
-            myPos.set(hero.getPosition());
-            life = hero.getLife();
+            Vector2 heroPos = hero.getHull().getBody().getWorldCenter();
```

Comment:
Where are we handling the "if hero is transcedent" condition from before?

Reply:
We need not, it was just a remains of the old pre-Hero code